Projects/App/Sources/Driver/Service/GifticonAPI.swift

```diff
@@ -43,6 +40,10 @@ extension GifticonAPI: NetworkRequestable {
             return nil
         case let .categoryList(model):
             return model
+        case .sprinkleApplication(_):
+            return nil
+        case .sprinkleDetail(_):
+            return nil
```

Comment:
```suggestion
        case .sprinkleApplication:
            return nil
        case .sprinkleDetail:
            return nil
```

사용하지 않는 associated value의 경우 위와같이 사용할 수 있습니다 !!

또한 case의 return이 같다면 아래와 같이 작성할수도 있습니다 😄 
```suggestion
        case .sprinkleApplication, .sprinkleDetail:
            return nil
```

Reply:
이와 관련된 스위치문을 모두 수정했습니다! 동일한 것은 묶어서 하니 더 가독성이 높아지는거 같네요 감사합니다!